plugins/NavStars/src/NavStars.cpp

```diff
@@ -469,7 +469,7 @@ void NavStars::extraInfo(StelCore* core, const StelObjectP& selectedObject, bool
 		.setJde(jde)
 		.setGmst(get_mean_sidereal_time(jd, jde));
 
-	StelUtils::rectToSphe(&x, &y, selectedObject->getEquinoxEquatorialPosApparent(core));	
+	StelUtils::rectToSphe(&x, &y, selectedObject->getEquinoxEquatorialPos(core));	
 	calc.setRaRad(x)
```

Comment:
Just to be clear on that: "...Apparent()" here means "including atmospheric refraction". To control this via atmosphere switch, you can use "...Auto()". Now refraction is switched off.

Reply:
Indeed, I didn't want refraction here at all which is why I didn't use Auto().